odoo/addons/base/tests/test_avatar_mixin.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from base64 import b64decode
+
+from odoo.tests.common import TransactionCase
+
+class TestAvatarMixin(TransactionCase):
+    """ tests the avatar mixin """
+    def setUp(self):
+        super(TestAvatarMixin, self).setUp()
+        self.partner_without_image = self.env['res.partner'].create({
+            'name': 'Marc Demo',
+            'email': 'mark.brown23@example.com',
+            'image_1920': False,
+            'create_date': '2015-11-12 00:00:00'
+        })
+        self.user_without_image = self.env['res.users'].create({
+            'login': 'demo_1',
+            'password': 'demo_1',
+            'partner_id': self.partner_without_image.id
+        })
```

Comment:
It's strange to manipulate `partner_id` of user.

Again here, could be created in a single step: just create the user with all the fields.

The partner is accessible from `self.user_without_image.partner_id`.

Reply:
If I write the `create_date` on the user, the create_date of the partner is not set as the same and the test fails